packages/api/src/providers/fitbit.ts

```diff
@@ -332,4 +337,68 @@ export class Fitbit extends Provider implements OAuth2 {
       }
     );
   }
+
+  async postAuth(token: string, userId?: string) {
+    const accessToken = JSON.parse(token).access_token;
+
+    const scopes = JSON.parse(token).scope;
+
+    const subscriptionTypes: Record<FitbitScopes, string> = {
+      [FitbitScopes.activity]: "activities",
+      [FitbitScopes.nutrition]: "foods",
+      [FitbitScopes.weight]: "body",
+      [FitbitScopes.sleep]: "sleep",
+    };
+
+    const subscriptionId = userId;
+
+    if (this.allRequiredScopesIncluded(scopes)) {
+      const subscriptionUrl = `${Fitbit.URL}/${Fitbit.API_PATH}/apiSubscriptions/${subscriptionId}.json`;
+      this.createSubscription(subscriptionUrl, accessToken);
+      console.log("All scopes included. User subscribed to all Fitbit WH collectionTypes.");
```

Comment:
as per my previous comment, do we really want to do this? this will subscribe to `userRevokedAccess` as well, which AFAIK we currently don't handle

perhaps we can leave this case out and make it a part of the TODO?

Reply:
Yes. Leaving this as TODO. Should be able to get back to this soon after some higher priority tasks.